### ISSUE TITLE
List supported Xcode versions on web page

### ISSUE THREAD

**Issue by fulldecent** [CocoaPods/CocoaPods#8238]
The CocoaPods project supports various versions of Xcode. And this version support is rolled in at some point, possibly not identical to Apple's release schedule. Please identify exactly which versions of Xcode are supported at these locations:

* https://cocoapods.org
* https://github.com/CocoaPods/CocoaPods/blob/master/README.md

**Comment by amorde** [comment]
I agree this could be improved. We should also probably list which OS / Ruby versions are supported as well.

**Comment by fulldecent** [comment]
At present, the website states this which implies Xcode 10 is NOT supported:

![screen shot 2018-11-20 at 12 02 23 am](https://user-images.githubusercontent.com/382183/48752507-9fac9e00-ec57-11e8-9c0d-2574a5ba8e1e.png)

I want to support Xcode 10 in my pod but I sure as hell don't have time to support a CocoaPods+Xcode combination that CocoaPods does not support.

**Comment by fulldecent** [comment]
At present, the website claims Xcode 9 and 10 are support. This implies Xcode 11 and 12 are not supported.

<img width="1421" alt="Screen Shot 2019-04-02 at 8 28 28 PM" src="https://user-images.githubusercontent.com/382183/55444764-f4591180-5585-11e9-867c-1d474562a8f6.png">

**Comment by orta** [comment]
Moving this to cocoapods.org - happy to take PRs fixing this
